stellargraph/datasets/datasets.py

```diff
@@ -850,25 +850,23 @@ def load(self):
         return StellarGraph(nodes=nodes, edges=edges, edge_weight_column="time"), edges
 
 
-@experimental(reason="the data isn't downloaded automatically", issues=[1303])
+@experimental(reason="tests and documentation missing", issues=[1303])
 class METR_LA(
     DatasetLoader,
     name="METR-LA",
     directory_name="METR-LA",
-    url="https://github.com/lehaifeng/T-GCN/tree/master/data",
-    url_archive_format="n/a",
-    expected_files=["los_speed.csv", "los_adj.csv"],
+    url="https://github.com/lehaifeng/T-GCN/archive/master.zip",
+    url_archive_format="zip",
+    url_archive_contains_directory=False,
```

Comment:
We could change `directory_name` to `T-GCN-master` and then remove this?

```suggestion
```

If we ever use the `sz` data from that repo, we could have the two datasets share a single downloaded folder.

Reply:
the disadvantage here is that the directory name in datasets is quite unintuitive

Reply:
That's fair, but I'm not sure it is a problem? I think it's fine for us to treat the `stellargraph-datasets` folder as an "unsupported" black box, where the format is totally internal? (This reasoning applies for the commit-hash-in-directory name below too.)

Reply:
I suppose ugly here is fine; most users won't even see it and it's not going to clash with another dataset!